MPF.md

```diff
@@ -1210,8 +1210,8 @@ For example:
 ### Configure periodic package inventory refresh interval
 
 CFEngine refreshes software inventory when it makes changes via packages
-promises. Additionally, by default, CFEngine periodically refreshes it's
-internal cache of packages installed (every 60 minutes) and package updates that
+promises. Additionally, by default, CFEngine refreshes it's
+internal cache of packages installed (during each agent) run and package updates that
```

Comment:
@nickanderson run should be inside parens.

Reply:
ACK, thanks!